### PR TITLE
Replace include macro with normal per country module files

### PR DESCRIPTION
**Pull request by AD4MANTIS**
Implemented the changes for #3 
It should work the same way as before, just restructured the code.
I never did any Python before so I'm open for feedback.

### CODE REVIEW

File: src/build.rs [hunk 1]
```diff
@@ -0,0 +1,518 @@
+use std::collections::HashSet;
+
+use crate::{data::*, prelude::*, HolidayMap, Result, Year};
+
+/// Generate holiday map for the specified countries and years.
+pub fn build(
```
Comment (yukinarit) [inline src/build.rs:6]: Can you merge src/builder.rs into src/build.rs?

**Review by yukinarit — COMMENTED** [review]
Hi @AD4MANTIS 

Thanks for big contribution!
I left some questions and comments, please take a look!

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,5 @@ Cargo.lock
 **/*.rs.bk
 
 poetry.lock
+
+python3Env
```
[the remark this answers is not in the document]
Comment (AD4MANTIS) [reply .gitignore]: Reverted. I used this for my pyhton3 enviroment.

File: src/data.rs
[the remark this answers is not in the document]
Comment (AD4MANTIS) [reply src/data.rs:1]: sure

File: src/build_help.rs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+use chrono::NaiveDate;
+
+use crate::{prelude::*, Holiday, HolidayPerCountryMap, Year};
+
+pub fn should_build_year(years: &Option<&std::ops::Range<Year>>, year: Year) -> bool {
+    years.is_none() || years.unwrap().contains(&year)
```
[the remark this answers is not in the document]
Comment (AD4MANTIS) [reply src/build_help.rs]: I used
```rs
years.map_or(true, |r| r.contains(&year))
```

File: src/build_help.rs [hunk 4]
```diff
@@ -0,0 +1,32 @@
+use chrono::NaiveDate;
```
[the remark this answers is not in the document]
Comment (AD4MANTIS) [reply src/build_help.rs]: You are right, i moved it as requested

File: src/build.rs (hunk 1 above)
Comment (AD4MANTIS) [reply src/build.rs:6]: I wouldn't merge these two into a singe file, because build.rs is generated, builder.rs is normal Rust code

**Review by yukinarit — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**